### DISCUSSION TITLE
AlgoSDK and myAlgoConnect version

### DISCUSSION THREAD

**Discussion by jchaselubitz** [reach-sh/reach-lang#1376] [Q&A] [answered]
Hello! How can we (@asolpshinning and I) determine which versions of AlgoSDK and myAlgoConnect Reach is using.

**Comment by jeapostrophe** [comment, marked as answer]
You can look at the package dependencies. `npm` should show you them, but you can also look at the source:

https://github.com/reach-sh/reach-lang/blob/master/js/js-deps/package.json#L28-L30
